drive-stack/net-detec/README.md

```diff
@@ -0,0 +1,39 @@
+# net-detec
+
+A tiny mDNS-SD **client/server** utility with a reusable library. It can **advertise** a service on a specific interface or **discover** services/hosts by service type or hostname.
```

Comment:
I don't understand what the purpose of this server is... What's this for?

Reply:
Please see [here](https://github.com/concordia-fsae/firmware/pull/341) for the `log-offloader` specifically. Its for non-critical applications to identify services on the local network and then interface with them